README.md

```diff
@@ -333,4 +333,354 @@ Other Style Guides
 
 **[⬆ back to top](#table-of-contents)**
 
+## Destructuring
+
+  - [5.1](#5.1) <a name='5.1'></a> Use object destructuring when accessing and using multiple properties of an object.
+
+  > Why? Destructuring saves you from creating temporary references for those properties.
+
+    ```javascript
+    // bad
+    function getFullName(user) {
+      const firstName = user.firstName;
+      const lastName = user.lastName;
+
+      return `${firstName} ${lastName}`;
+    }
+
+    // good
+    function getFullName(obj) {
+      const { firstName, lastName } = obj;
+      return `${firstName} ${lastName}`;
+    }
+
+    // best
+    function getFullName({ firstName, lastName }) {
```

Comment:
> Since you're destructing the object very literally using the braces function({foo, bar}, ...otherArguments), I don't feel this way. In the past it was always obvious if someone did it.

Good point, this is enough for me to agree :) anyway we will need some time to enforce it!